common/datasets/switchboard/corpus_train.py

```diff
@@ -0,0 +1,179 @@
+"""
+Functions related to speaker and training corpus creation
+"""
+__all__ = [
+    "get_train_bliss_corpus_ldc",
+    "get_train_bliss_corpus_i6_legacy",
+    "get_train_corpus_object_ldc",
+    "get_train_corpus_object_i6_legacy",
+]
+
+import os
+from sisyphus import tk
+
+from i6_core.tools.download import DownloadJob
+from i6_core.datasets.switchboard import (
+    DownloadSwitchboardTranscriptionAndDictJob,
+    CreateSwitchboardBlissCorpusJob,
+    SwitchboardSphereToWaveJob,
+    DownloadSwitchboardSpeakersStatsJob,
+    CreateSwitchboardSpeakersListJob,
+    CreateLDCSwitchboardSpeakerListJob,
+)
+
+from i6_experiments.common.datasets.util import CorpusObject
+
+from .constants import SUBDIR_PREFIX, durations
+from .paths import SWITCHBOARD1_PATH, SWITCHBOARD1_LEGACY_PATH
+
+
+def get_train_bliss_corpus(
+    audio_dir: tk.Path,
+    skip_empty_ldc_file: bool = False,
+    subdir_prefix: str = SUBDIR_PREFIX,
+) -> tk.Path:
+    """
+    Returns Switchboard training bliss corpus
+
+    :param audio_dir: path for audio data
+    :param subdir_prefix: alias name prefix
+    :param skip_empty_ldc_file: exclude sw2167B which is empty in the official LDC download
+    :return: Path to switchboard training corpus
+    """
+    swb_trans_and_dict = DownloadSwitchboardTranscriptionAndDictJob()
+    swb_trans_and_dict.add_alias(
+        os.path.join(subdir_prefix, "download_trans_and_dict_job")
+    )
+
+    speakers_list = get_speakers_list_legacy(subdir_prefix=subdir_prefix)
+    corpus = CreateSwitchboardBlissCorpusJob(
+        audio_dir=audio_dir,
+        trans_dir=swb_trans_and_dict.out_trans_dir,
+        speakers_list_file=speakers_list,
+        skip_empty_ldc_file=skip_empty_ldc_file,
+    )
+    corpus.add_alias(
+        os.path.join(
+            subdir_prefix,
+            "create_train_corpus_job_skip_empty_%s" % str(skip_empty_ldc_file),
+        )
+    )
+
+    return corpus.out_corpus
+
+
+def get_train_bliss_corpus_ldc(subdir_prefix: str = SUBDIR_PREFIX) -> tk.Path:
+    """
+    Switchboard-1 training corpus based on the original LDC file
+    Uses i6-custom text processing, and the transcriptions are fully
+    identical to the "i6-legacy" version.
```

Comment:
I am not sure but is this the right documentation. I understood `_ldc` is the new corpora and `_i6_legacy` is the "inherited" corpora.

Reply:
Yes, but this is not conflicting with this text, is it? Maybe you can say how to rewrite this if it is not clear... maybe I write "transcription format" instead of "transctriptions"...

Reply:
I would add something what is actually different to the i6-legacy version. Otherwise it seems that this is the same as i6-legacy

Reply:
It's also not intuitively clear to me which version is what exactly from looking at the docstrings.